kolibri/core/assets/src/utils/i18n.js

```diff
@@ -175,25 +175,82 @@ export function setUpVueIntl() {
     return $trWrapper(nameSpace, this.$options.$trs, this.$formatMessage, messageId, args);
   };
 
-  if (global.languageCode) {
-    currentLanguage = global.languageCode;
-    vue.setLocale(currentLanguage);
-    setLanguageDensity(currentLanguage);
+  vue.setLocale(currentLanguage);
+  if (global.coreLanguageMessages) {
+    vue.registerMessages(currentLanguage, global.coreLanguageMessages);
+  }
+  importVueIntlLocaleData().forEach(localeData => VueIntl.addLocaleData(localeData));
```

Comment:
Yeah, this was the change where I loaded everything synchronously because it was faster than loading one thing async.

In a future PR I'd like to the data from a `<link>` like I'm doing with the CSS

Reply:
follow-up issue: https://github.com/learningequality/kolibri/issues/4364